ci/ray_ci/macos/macos_ci.sh

```diff
@@ -11,31 +11,36 @@ export LANG="en_US.UTF-8"
 export BUILD="1"
 export DL="1"
 
+filter_out_flaky_tests() {
+  bazel run ci/ray_ci/automation:filter_out_flaky_tests -- "darwin:"
+}
```

Comment:
could you get all the other files into a PR (or reuse this PR, and revert this file, with the content saved on another branch)? and I can approve and merge all the python parts now.

Reply:
I've saved the content of this file on branch `khluu/split_flaky_tests_macos_suites`. Going to revert it now.

Reply:
Just reverted it.